### PR TITLE
Python 3: fix HomePlugAv

### PR DESCRIPTION
**Pull request by gpotter2**

### CODE REVIEW

File: scapy/fields.py [hunk 1]
```diff
@@ -380,6 +380,8 @@ def __init__(self, name, default):
 class StrField(Field):
     __slots__ = ["remain"]
     def __init__(self, name, default, fmt="H", remain=0):
+        if isinstance(default, str) and default == "":
```
Comment (p-l-) [inline scapy/fields.py]: I don't get it. Why would you make a special case for the empty string? I think we should fix all the default values (or add this hack for any `str` values, but I prefer the former).

Comment (gpotter2) [reply scapy/fields.py]: Hmmm, not sure why i did that....

**Comment by gpotter2** [comment]
@p-l- Could you rename (using the online github tool) `HomePlugAV.py` to `homePlugAV.py` (or anything starting not by a cap letter). Same for the .uts file.
It breaks the order of commands such as `list_contrib`

**Comment by p-l-** [comment]
@gpotter2 will do, when this PR is merged (Travis...). Please remind me if I forget :)

**Pull request merged**